### PR TITLE
Log Ceph status in case drain command times out

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/node.py [hunk 1]
```diff
@@ -151,12 +152,19 @@ def drain_nodes(node_names):
...
-        f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-        f"--delete-local-data", timeout=1200
-    )
+    try:
+        ocp.exec_oc_cmd(
+            f"adm drain {node_names_str} --force=true --ignore-daemonsets "
+            f"--delete-local-data", timeout=1200
+        )
+    except TimeoutExpired:
+        log.info(
```
Comment (RazTamir) [inline ocs_ci/ocs/node.py]: log.error?

Comment (ebenahar) [reply ocs_ci/ocs/node.py]: ack

File: ocs_ci/ocs/resources/pod.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
 from ocs_ci.ocs.ocp import OCP, verify_images_upgraded
 from tests import helpers
 from ocs_ci.ocs import workload
-from ocs_ci.ocs import constants, defaults, node
+from ocs_ci.ocs import constants, defaults
```
Comment (vasukulkarni) [inline ocs_ci/ocs/resources/pod.py]: Is this change related ? If not can it be moved to its own PR with issue?

Comment (ebenahar) [reply ocs_ci/ocs/resources/pod.py]: not needed, will revert
